Improve some readmes

Timonwa opened the pull request.

Royal-lobster commented:
@Timonwa looks good! but we also need to update our docs with the content in old contribution docs i think, then need to link it to the new contribution docs

Softdev1 commented:
> _ No description provided. _

at the minimum add a little  description to the pr request

Softdev1 commented:
@gemini-code-assist review

Timonwa commented:
Hi @Royal-lobster, I have renamed the old contributing guide to `ARCHITCEHTURE.md` pending when it is moved to the doc app. Then I added a backlink to it in the new contribution.md. We should be able to close this pr now.
